.docker/paste/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM rustlang/rust:nightly
+
+WORKDIR /app
+
+RUN apt update -y && apt install -y libsodium-dev cmake build-essential postgresql postgresql-client
+
+# TODO: get it from .env instead? but then it's not accessable by the container.
+COPY postgres_env /root/postgres_env
+RUN sed -E 's/[^=]+=//g' /root/postgres_env | tr "\n" " " | awk '{ print "db:5432:" $1 ":" $2 ":" $3 }' > /root/.pgpass && chmod 0600 /root/.pgpass
+RUN shred /root/postgres_env && rm /root/postgres_env
```

Comment:
Is `shred` really necessary? Doesn't Docker keep layers that will have this file in them?

Reply:
necessary, no, but it's not like it is an expensive operation.